cmd/ci-secret-bootstrap/main.go

```diff
@@ -761,8 +761,14 @@ func validateBWItems(client secrets.ReadOnlyClient, secretConfigs []secretbootst
 	var errs []error
 
 	for _, config := range secretConfigs {
-		for _, item := range config.From {
+		if config.Generated {
```

Comment:
/hold
This should be possible without additional config opts, ref https://coreos.slack.com/archives/GB7NB0CUC/p1619441793136000?thread_ts=1619440512.133900&cid=GB7NB0CUC